sdk/ai/azure-ai-agents/samples/agents_tools/sample_agents_computer_use.py

```diff
@@ -0,0 +1,205 @@
+    MessageInputImageUrlBlock,
+    RequiredComputerUseToolCall,
+    SubmitToolOutputsAction
+)
+from azure.identity import DefaultAzureCredential
+
+def image_to_base64(image_path: str) -> str:
+    """
+    Convert an image file to a Base64-encoded string.
+
+    :param image_path: The path to the image file (e.g. 'image_file.png')
+    :return: A Base64-encoded string representing the image.
+    :raises FileNotFoundError: If the provided file path does not exist.
+    :raises OSError: If there's an error reading the file.
+    """
+    if not os.path.isfile(image_path):
+        raise FileNotFoundError(f"File not found at: {image_path}")
+
+    try:
+        with open(image_path, "rb") as image_file:
+            file_data = image_file.read()
+        return base64.b64encode(file_data).decode("utf-8")
+    except Exception as exc:
+        raise OSError(f"Error reading file '{image_path}'") from exc
+
+asset_file_path = os.path.abspath(os.path.join(os.path.dirname(__file__), "../assets/cua_screenshot.jpg"))
+action_result_file_path = os.path.abspath(os.path.join(os.path.dirname(__file__), "../assets/cua_screenshot_next.jpg"))
+project_client = AIProjectClient(endpoint=os.environ["PROJECT_ENDPOINT"], credential=DefaultAzureCredential())
+
+# Initialize Computer Use tool with a browser-sized viewport
+environment = os.environ.get("COMPUTER_USE_ENVIRONMENT", "windows")
+computer_use = ComputerUseTool(display_width=1026, display_height=769, environment=environment)
+
+with project_client:
+
+    agents_client = project_client.agents
+
+    # Create a new Agent that has the Computer Use tool attached.
+    agent = agents_client.create_agent(
+        model=os.environ["MODEL_DEPLOYMENT_NAME"],
+        name="my-agent-computer-use",
+        instructions="""
+            You are an computer automation assistant. 
+            Use the computer_use_preview tool to interact with the screen when needed.
+            """,
+        tools=computer_use.definitions,
+    )
+
+
+    print(f"Created agent, ID: {agent.id}")
+
+    # Create thread for communication
+    thread = agents_client.threads.create()
+    print(f"Created thread, ID: {thread.id}")
+
+    input_message = ("I can see a web browser with bing.com open and the cursor in the search box."
+                     "Type 'movies near me' without pressing Enter or any other key. Only type 'movies near me'.")
+    image_base64 = image_to_base64(asset_file_path)
+    img_url = f"data:image/jpeg;base64,{image_base64}"
+    url_param = MessageImageUrlParam(url=img_url, detail="high")
+    content_blocks: List[MessageInputContentBlock] = [
+        MessageInputTextBlock(text=input_message),
+        MessageInputImageUrlBlock(image_url=url_param),
+    ]
+    # Create message to thread
+    message = agents_client.messages.create(
+        thread_id=thread.id,
+        role=MessageRole.USER,
+        content=content_blocks
+    )
+    print(f"Created message, ID: {message.id}")
+
```

Comment:
As part of your manual tests, please turn on AzureMonitor tracing with content (`set OTEL_INSTRUMENTATION_GENAI_CAPTURE_MESSAGE_CONTENT=ture`) to make sure this payload does not break tracing.